app/graphql/mutations/repository/remove_url_mapping_mutation.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+module Mutations
+  module Repository
+    RemoveUrlMappingMutation = GraphQL::Field.define do
+      type types.Boolean
+      description 'Removes an URL Mapping'
+
+      argument :repositoryId, !types.ID do
+        description 'The ID of the repository in which the URL mapping exits'
+      end
+
+      argument :urlMappingId, !types.ID do
+        description 'The ID of the URL mapping'
+      end
+
+      resource!(lambda do |_root, arguments, _context|
+        ::Repository.first(slug: arguments[:repositoryId])
+      end)
+
+      authorize! :update
+
+      resolve RemoveUrlMappingResolver.new
+    end
+
+    # GraphQL mutation to add new URL mappings
+    class RemoveUrlMappingResolver
+      def call(repository, arguments, context)
+        url_mapping = repository.url_mappings_dataset.
+          first(id: arguments[:urlMappingId])
+        if url_mapping.nil?
```

Comment:
I think this is not needed. Since the goal is to remove the mapping, if the mapping wasn't there in the first place, we still achieved what we wanted.
You can just do
```ruby
url_mapping.destroy if url_mapping
true
```